### PR TITLE
Loompy recipe

### PR DESCRIPTION
**Pull request by gioelelm**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/loompy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+source:
+  fn: '{{ name }}-{{ version }}.tar.gz'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  skip: True  # [not py36]
+  number: 0
+  script: python setup.py install  --single-version-externally-managed --record=record.txt
```
Comment (bgruening) [inline recipes/loompy/meta.yaml]: If you include the this line, you don't need the script file.

Comment (gioelelm) [reply recipes/loompy/meta.yaml]: Ok, should I remove the script? Should I just commit the change in the forked repo or make a new pull request?

Comment (bgruening) [reply recipes/loompy/meta.yaml]: Yes please remove the build.sh script and commit it into the same branch, the PR will then be updated automatically.

Comment (gioelelm) [reply recipes/loompy/meta.yaml]: Done.
A question slightly OT, now that python3.7 is being released, probably [not py36] is not a good preprocessing tag to mean python>=3.6, right ? maybe [py < "36"] or [int(py) < 36] are more correct?

Comment (bgruening) [reply recipes/loompy/meta.yaml]: Normally you just write [py36 or py37]. Not sure if your examples will work, never tried it.

**Pull request merged**